packages/flutter/test/material/navigation_bar_test.dart

```diff
@@ -1145,11 +1145,13 @@ void main() {
     await gesture.moveTo(tester.getCenter(find.byType(NavigationIndicator).last));
     await tester.pumpAndSettle();
 
-    final RenderObject inkFeatures = tester.allRenderObjects.firstWhere((RenderObject object) => object.runtimeType.toString() == '_RenderInkFeatures');
+    // This test extracts the controller from the allRenderObjects iterable
+    // in order to prevent some wonkiness when running via skwasm.
+    final RenderObject controller = tester.allRenderObjects.firstWhere((RenderObject object) => object is SplashController);
```

Comment:
The `Material.of` approach was failing on skwasm?

Reply:
Yeah, it worked fine when I ran the test locally, but GitHub CI was not happy 😓